server/ec2spotmanager/templates/config/view.html

```diff
@@ -3,35 +3,8 @@
-			<a href="{% url 'ec2spotmanager:configdel' config.pk %}" class="btn btn-danger">Delete</a>
-			<a href="{% url 'ec2spotmanager:configcycle' config.pk %}" class="btn btn-danger">Cycle pools using this configuration</a>
-	</div>
-	<table class="table table-condensed table-hover table-bordered">
-		<tr><td>Name</td><td>{{ config.name }}</td></tr>
-		<tr><td>Parent</td><td>
-			{% if config.parent %}
-				<a href="{% url 'ec2spotmanager:configview' config.parent.pk %}">{{ config.parent.name }}</a>
-				{% else %}
-				None
-			{% endif %}
-		</td></tr>
-		<tr><td>Size (cores)</td><td>{% if config.size %} {{ config.size }} {% else %} Not specified {% endif %}</td></tr>
-		<tr><td>Cycle Interval</td><td>{% if config.cycle_interval %} {{ config.cycle_interval }} seconds {% else %} Not specified {% endif %}</td></tr>
-		<tr><td>EC2 Key Name</td><td>{% if config.ec2_key_name %} {{ config.ec2_key_name }} {% else %} Not specified {% endif %}</td></tr>
-		<tr><td>EC2 Security Groups</td><td>{% if config.ec2_security_groups_list %} {{ config.ec2_security_groups_list|listcsv }} {% else %} Not specified {% endif %}{% if config.ec2_security_groups_override %} (overwrite parent) {% endif %}</td></tr>
-		<tr><td>EC2 Instance Types</td><td>{% if config.ec2_instance_types_list %} {{ config.ec2_instance_types_list|listcsv }} {% else %} Not specified {% endif %}{% if config.ec2_instance_types_override %} (overwrite parent) {% endif %}</td></tr>
-		<tr><td>EC2 Image Name</td><td>{% if config.ec2_image_name %} {{ config.ec2_image_name }} {% else %} Not specified {% endif %}</td></tr>
-		<tr><td>EC2 Userdata File</td><td>{% if config.ec2_userdata_file %} {{ config.ec2_userdata_file }} {% else %} Not specified {% endif %}</td></tr>
-		<tr><td>EC2 Userdata Macros</td><td>{% if config.ec2_userdata_macros_dict %} {{ config.ec2_userdata_macros_dict|dictcsv }} {% else %} Not specified {% endif %}{% if config.ec2_userdata_macros_override %} (overwrite parent) {% endif %}</td></tr>
-		<tr><td>EC2 Allowed Regions</td><td>{% if config.ec2_allowed_regions_list %} {{ config.ec2_allowed_regions_list|listcsv }} {% else %} Not specified {% endif %}{% if config.ec2_allowed_regions_override %} (overwrite parent) {% endif %}</td></tr>
-		<tr><td>EC2 Maximum Price (per core)</td><td>{% if config.ec2_max_price %} {{ config.ec2_max_price }} USD {% else %} Not specified {% endif %}</td></tr>
-		<tr><td>EC2 Additional Tags</td><td>{% if config.ec2_tags_dict %} {{ config.ec2_tags_dict|dictcsv }} {% else %} Not specified {% endif %}{% if config.ec2_tags_override %} (overwrite parent) {% endif %}</td></tr>
-		<tr><td>EC2 Additional Raw Configuration</td><td>{% if config.ec2_raw_config_dict %} {{ config.ec2_raw_config_dict|dictcsv }} {% else %} Not specified {% endif %}{% if config.ec2_raw_config_override %} (overwrite parent) {% endif %}</td></tr>
-	</table>
+	{% if configuration == configuration.provider.ec2configuration %}
```

Comment:
`configuration.provider.ec2configuration` is old code from the class-based inheritance change. This does nothing anymore.